components/brave_today/browser/feed_building.cc

```diff
@@ -248,6 +248,16 @@ bool ShouldDisplayFeedItem(const mojom::FeedItemPtr& feed_item,
             << data->publisher_id << ": " << publisher->publisher_name;
     return false;
   }
+
+  // Direct publishers should be shown, even though they aren't in any locales,
+  // and their enabled status is |NOT_MODIFIED|.
+  if (publisher->type == brave_news::mojom::PublisherType::DIRECT_SOURCE) {
```

Comment:
IIRC, we don't need to check direct source's `UserEnabled` is ENABLED or not because it's deleted from list when it's disabled?

Reply:
Yeah, exactly!